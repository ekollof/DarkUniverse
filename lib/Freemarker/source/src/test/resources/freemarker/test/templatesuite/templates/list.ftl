<#assign animals = ["aardvark", "bear", "cat", "dog", "elephant"]>

<@testSeq animals />

<@testSeq arrayList />

<@testSeq linkedList />

<@testSeq set />

<@testSeq iterator />

<#macro testSeq seq>
Size: <#attempt>${seq?size}<#recover>failed</#attempt>
Items: <#list seq as i>@${i_index} ${i}<#if i_has_next>, <#else>.</#if></#list>
</#macro>