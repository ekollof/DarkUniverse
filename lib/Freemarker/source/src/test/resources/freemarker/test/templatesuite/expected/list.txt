
Size: 5
Items: @0 aardvark, @1 bear, @2 cat, @3 dog, @4 elephant.

Size: 3
Items: @0 11, @1 22, @2 33.

Size: 3
Items: @0 11, @1 22, @2 33.

Size: 3
Items: @0 11, @1 22, @2 33.

Size: failed
Items: @0 11, @1 22, @2 33.

